plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -146,7 +147,8 @@ public MachineProviderImpl(DockerConnector docker,
                                @Named("machine.docker.machine_env") Set<String> allMachinesEnvVariables,
                                @Named("che.docker.registry_for_snapshots") boolean snapshotUseRegistry,
                                @Named("che.docker.swap") double memorySwapMultiplier,
-                               @Named("machine.docker.networks") Set<Set<String>> additionalNetworks)
+                               @Named("machine.docker.networks") Set<Set<String>> additionalNetworks,
+                               @Nullable @Named("che.machine.docker.volumes.projects.options") String volumeOptions)
```

Comment:
We use dots to separate categories of implementation. We use underscores to separate words of single term. Please ensure that you are following that policy